runtime/container_linux.go

```diff
@@ -180,6 +180,7 @@ func (c *container) getMemoryEventFD(root string) (*oom, error) {
 	if serr != 0 {
 		return nil, serr
 	}
+	syscall.CloseOnExec(int(fd))
```

Comment:
Ok, I found the issue, please replace the flag in `RawSyscall` with `unix.EFD_CLOEXEC`